devito/symbolics/extended_sympy.py

```diff
@@ -701,6 +696,31 @@ def rfunc(func, item, *args):
 }
 
 rfunc_mapper = {
-    min: MIN,
-    max: MAX,
+    min: Min,
+    max: Max,
 }
+
+
+def integer_args(*args):
```

Comment:
If this function is used just for dispatching to the correct printing routing, should we not keep it inside `printer.py`?

Reply:
Yeah couldn't decide happy either way

Reply:
moved